DMTN-276.tex

```diff
@@ -53,88 +56,263 @@
-  }\label{fig:ex_persistence}
+  Full Focal plane images with amplifier level resoltuion comparing a subsequent dark image with nomial parallel clock swing voltages (left; Run 13392) 
+  and one with a slightly higher parallel clock swing voltage (right; Run 13410).
+  % This particular flash has one of the three spots inbetween two amplifiers and you can see the trail starting at the spots and going all the way to the readout of each amplifier.
+  }\label{fig:Flat_Induced_Comparison}
 \end{figure*}
 
+One of the concerning discoveries with these measurements were the level of persistence was reaching 10 ADU on a per amp basis.
+Figure \ref{fig:Flat_Induced} shows an example of a focal plane plot displaying the per amp value of the first dark image after the flash as well as the actual dark image.
+One of the ways to lower or remove the persistence effect is to adjust the parallel swing voltages of the detector.
+This was tested and Figure \ref{fig:Flat_Induced_Comparison} comapares the per amp plot between the nominal voltages and the slightly higher parallel clock swing voltages (-9.3 vs -9.0 V).
+This change did lower the persistence level but much less than what was expected (only $10\%$).
+
+
+\subsubsection{Spot Induced Persistence}
+During Run 6b, we utilized the pinhole filter to measure the persistence of the E2V sensors.
+The pinhole filter is positioned so that a small area of light hits one amplifier on each of the middle detectors on a raft.
+This means that we can only measure the persistence of 13 e2V detectors but we are able to get measurements across the focal plane.
+For these tests, we started with flashes from around 20 ke-/pixel below full well (~100 ke-/pixel) and ramped to twice full well (200 ke-/pixel).
+After each flash, we take 10, 15 second dark images which we use to measure the persistence.
+These images are then overscan and bias corrected.
+Figure \ref{fig:Run6_example} shows an example of one of the saturated pixels (spot) and the subsquent dark image with persistence.
+
 \begin{figure*}[!htp]
   \centering
-  \includegraphics[width=0.95\textwidth, angle=0]{DC2_percent_affected_pixels.png}
+  \includegraphics[width=0.85\textwidth, angle=0]{Run6_ex.png}
+  \caption{
+  Similar to Figure \ref{fig:ex_persistence_Run5} but with the pinhole filter. 
+  This shows the initail flash and the spot (left) and the dark imgaes that followed.
+  }\label{fig:Run6_example}
+\end{figure*}
+
+
+Using these images, we are able to measure the persistence induced by the saturated pixels as well as the trail that follows.
+We then model the persistence decay using this function:
+\begin{equation*}
+  Counts= A * \exp(t/\tau)+c
+\end{equation*}
+where \textit{Counts} are the average number of counts at the location of the spot in the following dark image, \textit{t} is the integration time since the flash, \textit{$\tau$} is the decay constant and \textit{A} and \textit{c} are constants.
```

Comment:
It is the wallclock time, as I am just taking the MJD from the images (I think this is set as the beginning of integration). I will change the text to say 'time since flash, 15 seconds of which are with the CCDs integrating'